# No Request
---
thesis/ObjectStamped[] objects
